tools/app-provisioning-tool/README.md

```diff
@@ -1,18 +1,30 @@
 # msidentity-app-sync 
 Command line tool that creates Microsoft identity platform applications in a tenant (AAD or B2C) and updates the configuration code of your ASP.NET Core applications (mvc, webapp, blazorwasm, blazorwasm hosted, blazorserver). The tool can also be used to update code from an existing AAD/AAD B2C application.
 
+
+
 ## Installing/Uninstalling the tool
 
-1. Build the repository and create the NuGet package (from the `tools\app-provisioning-tool` folder):
- 
-   ```Shell
-   dotnet pack
-   ```
-   
-2. Run the following in a developer command prompt in the `tools\app-provisioning-tool` folder:
+You have two ways of installing the CLI tool:
+- either from NuGet
+- or build it and install it from the repository
+
+### Option 1: Install from NuGet
+
+The following command will install the tool from NuGet.org
+
+```Shell
+dotnet tool install --global msidentity-app-sync
    
-   ```Shell
-   dotnet tool install --global --add-source app-provisioning-tool\nupkg msidentity-app-sync
+### Option 2: build and install from the repository
+
+You can clone the repository, build the tool and install it
+
+```Shell
+git clone https://github.com/AzureAD/microsoft-identity-web
+cd tools\app-provisionning-tool
+dotnet pack
+dotnet tool install --global --add-source app-provisioning-tool\nupkg msidentity-app-sync
    ```
 
 If later you want to uninstall the tool, just run (from anywhere):
```

Comment:
```suggestion
### Uninstalling the tool

If later you want to uninstall the tool, just run (from anywhere):
```

Reply:
@sayedihashimi I've provided suggestions, some of which I've already committed.
I'm going to approve this PR now and merge